crates/conversions/tests/e2e/dec_string.rs

```diff
@@ -0,0 +1,111 @@
+#[cfg(test)]
+mod tests_short_string {
+    use cairo_felt::Felt252;
+    use conversions::FromConv;
+    use starknet::core::types::FieldElement;
+    use starknet_api::core::{ClassHash, ContractAddress, Nonce};
+    use starknet_api::hash::{StarkFelt, StarkHash};
+
+    #[test]
+    fn test_short_strings_conversions_happy_case() {
+        let short_string = "1".to_string();
+
+        assert_eq!(
+            short_string,
+            String::from_(ClassHash::from_(short_string.clone()))
+        );
+        assert_eq!(
+            short_string,
+            String::from_(ContractAddress::from_(short_string.clone()))
+        );
+        assert_eq!(
+            short_string,
+            String::from_(Felt252::from_(short_string.clone()))
+        );
+        assert_eq!(
+            short_string,
+            String::from_(FieldElement::from_(short_string.clone()))
+        );
+        assert_eq!(
+            short_string,
+            String::from_(Nonce::from_(short_string.clone()))
+        );
+        assert_eq!(
+            short_string,
+            String::from_(StarkFelt::from_(short_string.clone()))
+        );
+        assert_eq!(
+            short_string,
+            String::from_(StarkHash::from_(short_string.clone()))
+        );
+    }
+
+    #[test]
+    fn test_short_strings_conversions_zero() {
+        let short_string = "0".to_string();
```

Comment:
Wouldn't it be cleaner to have just 3 cases for one test?

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/1208
New issue since it's out of scope for this one